### PR TITLE
driver-adapters: convert Decimals to Strings when calling JS

### PR DESCRIPTION
**Pull request by SevInf**
When forwarding the input to the driver adapter, we used to convert
decimal values to f64. On a large numbers, that lost some precision.
Changing this to convert them to strings instead.

Fix prisma/team-orm#497

TODO: after this gets to the client, unskip `decimal/presion` test
there.

### CODE REVIEW

**Review by Jolg42 — APPROVED** [review]

File: query-engine/driver-adapters/src/conversion.rs [hunk 1]
```diff
@@ -75,11 +75,8 @@ pub fn values_to_js_args(values: &[quaint::Value<'_>]) -> serde_json::Result<Vec
                 Some(bytes) => JSArg::Buffer(bytes.to_vec()),
                 None => JsonValue::Null.into(),
             },
-            quaint_value @ quaint::ValueType::Numeric(bd) => match bd {
-                Some(bd) => match bd.to_string().parse::<f64>() {
-                    Ok(double) => JSArg::from(JsonValue::from(double)),
-                    Err(_) => JSArg::from(JsonValue::from(quaint_value.clone())),
-                },
+            quaint::ValueType::Numeric(bd) => match bd {
+                Some(bd) => JSArg::RawString(bd.to_string()),
```
Comment (jkomyno) [inline query-engine/driver-adapters/src/conversion.rs:79]: Can we leave a comment here briefly explaining the rationale of this line for future reference?

File: query-engine/connector-test-kit-rs/query-engine-tests/tests/writes/data_types/native_types/mysql.rs [hunk 2]
```diff
@@ -198,6 +198,35 @@ mod mysql {
...
+              }
+            ) {
+              decLarge
+            }
+          }"#),
+          @r###"{"data":{"createOneModel":{"decLarge":"131603421.38724228"}}}"###
+        );
+
+        Ok(())
+    }
```
Comment (jkomyno) [inline query-engine/connector-test-kit-rs/query-engine-tests/tests/writes/data_types/native_types/mysql.rs:228]: I'm not sure why we're only testing MySQL and Vitess here?

**Review by jkomyno — APPROVED** [review]

**Comment by jkomyno** [comment]
Trying this PR locally, I verified that `decimal/precision` can indeed be skipped in functional Client tests.
Let's see if we can achieve this without breaking LibSQL. Remember: we might just add custom logic to the Driver Adapter to simplify our Rust logic (that was our original goal, I'm not sure whether that's still what we're pursuing though)

**Comment by SevInf** [comment]
Libsql is now fixed (to the extend it works on main, so 2 failed test)

**Pull request merged**